hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -196,6 +196,24 @@ public static void verifyResourceName(String... resourceNames) {
     }
   }
 
+  /**
+   * verifies that key name is a valid name.
+   *
+   * @param keyName key name to be validated
+   *
+   * @throws IllegalArgumentException
+   */
+  public static void verifyKeyName(String keyName) {
+    if (keyName == null) {
+      throw new IllegalArgumentException("Key name is null");
+    }
+    if(!OzoneConsts.KEYNAME_ILLEGAL_CHARACTER_CHECK_REGEX
+            .matcher(keyName).matches()){
+      throw new IllegalArgumentException("key name contains " +
+              "illegal characters.");
```

Comment:
I would like to propose making this message consistent with the one in `OmUtils`, so that `ozone fs -put` and `ozone sh key put` both print the same error.  I think it can be most simply done by moving the message from `OmUtils` here and letting the `OMException` propagate the underlying exception's message.

```suggestion
      throw new IllegalArgumentException("Invalid key name: " + keyName);
```

If you agree, please feel free to wait for others' reviews before making this change, to reduce the number of rounds.

Reply:
Thanks for the suggestion, I agree with you. 
I'll wait for others' reviews and suggestions before I make this change.